docs/en_US/TrialExample/Pix2pixExample.rst

```diff
@@ -0,0 +1,120 @@
+
+* The trial code for this example is adapted from the `repository maintained by the authors of Pix2pix and CycleGAN <https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix>`__ .  You can also use your previous code directly, refer to `How to define a trial <Trials.rst>`__ for modifying the code.
+* By default, the code uses the dataset "facades". It also supports the datasets 'night2day', 'edges2handbags', 'edges2shoes', and 'maps'.
+* For 'facades', 200 epochs are enough for the model to converge to a point where the difference between different parameters are salient enough for evaluation. If you are using other datasets, please consider increasing the ``n_epochs`` and ``n_epochs_decay`` parameters by either passing them as arguments when calling ``pix2pix.py`` in the config file (discussed below) or changing the ``pix2pix.py`` directly.
+* In this example, we use L1 loss on the test set as the score to report to NNI. Although L1 is by no means a comprehensive measure of image generation performance, at most times it makes sense for evaluating pix2pix models with similar hyperparameter settings. 
+
+
+**Config**
+
+Here is the example of running this experiment on local (with a single GPU):
+
+code directory: :githublink:`examples/trials/pix2pix-pytorch/config_v2.yml <examples/trials/pix2pix-pytorch/config_v2.yml>`
+
+Although using the new config version is recommended, it is also possible to run the experiment in the config v1 format:
+
+code directory: :githublink:`examples/trials/pix2pix-pytorch/config.yml <examples/trials/pix2pix-pytorch/config.yml>`
+
+To have a full glance on examples we have implemented our implementations: :githublink:`examples/trials/pix2pix-pytorch/ <examples/trials/pix2pix-pytorch>`
+
+Launch the experiment
+^^^^^^^^^^^^^^^^^^^^^
+
+We are ready for the experiment, let's now **run the config_v2.yml file from your command line to start the experiment**.
+
+.. code-block:: bash
+
+   nnictl create --config nni/examples/trials/pix2pix-pytorch/config_v2.yml
+
+Collecting the Results
+^^^^^^^^^^^^^^^^^^^^^^
+
+By default, our trial code saves the final trained model for each trial in the ``checkpoints/`` directory in the trial directory of the NNI experiment. The ``latest_net_G.pth`` and ``latest_net_D.pth`` correspond to the save checkpoints for the generator and the discriminator.
+
+To make it easier to run inference and see the generated images, we also incorporate a simple inference code here: :githublink:`test.py <examples/trials/pix2pix-pytorch/test.py>`
+
+To use the code, run the following command:
+
+.. code-block:: bash
+
+   python3 test.py -c CHECKPOINT -p PARAMETER_CFG -d DATASET_NAME -o OUTPUT_DIR
+
+``CHECKPOINT`` is the directory saving the checkpoints (e.g., the ``checkpoints/`` directory in the trial directory). ``PARAMETER_CFG`` is the ``parameter.cfg`` file generated by NNI recording the hyperparameter settings. This file can be found in the trial directory created by NNI. 
```

Comment:
Is performance available? Or is there any insights on which hyper-parameter is better?

Reply:
I do have a couple of experiments. I will add the results in a later commit.